package.json

```diff
@@ -51,6 +51,13 @@
         "Comment": "Unofficial client for Microsoft Teams for Linux",
         "StartupWMClass": "teams-for-linux"
       },
+      "extraResources": [
+        {
+          "from": "node_modules/spellchecker/vendor",
```

Comment:
Hi @noderat ,

Thanks a lot for your contribution!

maybe we should be adding the `/usr/share/hunspell/` instead so it picks up the ones installed in your Linux.

Not a big deal, just so then all the "linking" steps aren't needed.

Can you update the https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/KNOWN_ISSUES.md with this extraResources magic?

Reply:
That would only work when building from source. Most users of this (including many within my organization) I imagine are not prepared to build from source, instead using the Arch package or AppImage. As an alternative it looks like there may be a way to allow dictionaries to be added dynamically but that'll involve a much larger code change.

I'll update this PR with the change to Known Issues to describe the option to tweak the path to pick up the system dictionaries when building from source.

Reply:
Hi, it seems like pr #119 will solve both issues and should, in theory, also work when its build. 

Because it also solves the hunspell issue (or looks like), and it should work when its build, I will probably push that one instead of this one. 

That is pretty odd to have two people solving the same issue almost at the same time in different ways. I really appreciate your contribution and can't thank you enough for it.